lib/class/class_userPreferences.dart

```diff
@@ -24,7 +26,8 @@ class UserPreferences {
   bool showAllWarnings = false;
   bool areWarningsFromCache = false;
 
-  String versionNumber = "0.5.1"; // shown in the about view
+  String versionNumber = "0.6.0 - Alpha2 "; // shown in the about view
+  String previousInstalledVersion  = "";
   // String githubVersionNumber = versionNumber; // used in the update check
```

Comment:
I don't remember if we discussed it. I am not sure. At the moment we don't activate the code so it can be deleted. But if I look at the number of downloads from Github, 66 people have downloaded the latest version from Github. So maybe we should use it again, maybe even with an automatic check in the background? I've now asked on Mastodon what people think. Maybe we'll get an answer there. What is your opinion?

Reply:
> people getting the app from GitHub are aware of the Android app named Obtainium

That is correct. I have removed the old code for now.